### ISSUE TITLE
3DFox OS

### ISSUE THREAD

**Issue by EggFox** [Universal-Team/db#92]
### Title

3DFox OS

### System(s)

3DS

### Categories

Luma3DS

### Image link

https://github.com/EggFox/3DFox-OS/blob/main/fox_007-rejghmovebg.png

### Icon link

https://github.com/EggFox/3DFox-OS/blob/main/fox_007-rejghmovebg.png

### Long description

an operating system for 3ds always based on the default firmware. the operating system has the purpose of making all the services online (still not all) such as youtube, miiverse, nintendo zone and even the beloved anime channel; we also have a game store and an app store: the hshop and of course your store! which I find very useful. the system also includes a new home menu and new icons.

### Screenshots

![HNI_0001](https://user-images.githubusercontent.com/109808626/187456459-50485784-61a6-4cc1-bcfb-7d5970b503b0.JPG)
![HNI_0002](https://user-images.githubusercontent.com/109808626/187456470-39026f39-530a-4118-9f1a-b9e157d138c5.JPG)
![HNI_0003](https://user-images.githubusercontent.com/109808626/187456475-b4122cee-3696-4ab9-aeae-7bf27f973e5b.JPG)
![HNI_0004](https://user-images.githubusercontent.com/109808626/187456477-76eaeab2-5728-4081-b3de-6c058b4b7bd7.JPG)
![HNI_0005](https://user-images.githubusercontent.com/109808626/187456481-54182506-1cb2-4e80-a51d-8b0fe34f0825.JPG)

### GitHub repository of the app

https://github.com/EggFox/3DFox-OS

### Author

EggFox

### Short description

system for 3ds

### Source code link

https://github.com/EggFox/3DFox-OS

### Creation date

22/07/22

### Last update date

10/08/22

### Website link

https://www.3dfoxos.ga

### Wiki link

https://github.com/EggFox/3DFox-OS

### Download page link

https://www.3dfoxos.ga/download

### Version number

3.0

### Version title

Version 3.0

### Download link(s)

https://www.3dfoxos.ga/download
https://github.com/EggFox/3DFox-OS/releases/tag/3.0

### Developer permission

- [X] I am the app developer or have permission from the app developer to make this request

**Comment by EggFox** [comment]
How is it installed?
Download The Desired Version

Extract Everything On Your SD Card

Start the Console in GoodMode9

Run 3DFox_OS_Install Script

Start the InstallMenuHome and InstallApps entry

For Lite Version Press Install

You Are Done !!! Visit The Site Frequently For Updates

**Comment by CyberYoshi64** [comment]
Um, I don't think it's a good idea.
- 3DiScord has been discontinued and the GBATemp pages of said projects claim to avoid it to prevent getting banned, as it's an unofficial client and probably very outdated.
- A known piracy homebrew is included, I don't need to say more.
- Putting several applications into the bundle is also not ideal, has to keep track of every of these apps getting updated, including the Universal Updater, which is in the bundle as well.
I see the idea, but it's just unlikely, seeing as it's not its own OS, like a mock OS or anything, but just a modified HOME Menu (likely only a default theme changed) and some homebrew apps.
Mentioning *inside* the installer that it's safe is also fishy at best.

**Comment by cooolgamer** [comment]
can you tell me why you are taking credit? I just want to know why, there is a reason for sure right?

**Comment by NightScript370** [comment]
1. All-in-ones are not acceptable unless all involved parties would like to advertise themselves as such, and I think cooolgamer can confirm you certainly didn't. Even then, we prefer all apps being separate unless it would flood our unistore (like the emulator pack thingy)
2. Each individual app needs to comply within our specifications, such as credit from the owner and not being a piracy app. You are violating two of them.

TL;DR: No.
